src/img_to_text_gen.py

```diff
@@ -0,0 +1,57 @@
+import torch
+from transformers import VisionEncoderDecoderModel, ViTFeatureExtractor, AutoTokenizer
+
+itt_model = None
+itt_feature_extractor = None
+itt_tokenizer = None
+itt_device = None
+itt_inited = False
+
+def init_itt():

```

Comment:
Basically, this is an attempt to make a singleton object. This is a good idea, but you need to do that adequately 😅